tcc/src/main/java/org/apache/seata/rm/tcc/RMHandlerTCC.java

```diff
@@ -16,23 +16,54 @@
  */
 package org.apache.seata.rm.tcc;
 
+import org.apache.seata.core.model.BranchStatus;
 import org.apache.seata.core.model.BranchType;
 import org.apache.seata.core.model.ResourceManager;
+import org.apache.seata.core.protocol.ResultCode;
+import org.apache.seata.core.protocol.transaction.BranchDeleteRequest;
+import org.apache.seata.core.protocol.transaction.BranchDeleteResponse;
 import org.apache.seata.core.protocol.transaction.UndoLogDeleteRequest;
+import org.apache.seata.integration.tx.api.fence.DefaultCommonFenceHandler;
 import org.apache.seata.rm.AbstractRMHandler;
 import org.apache.seata.rm.DefaultResourceManager;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * The type Rm handler tcc.
  *
  */
 public class RMHandlerTCC extends AbstractRMHandler {
+    private static final Logger LOGGER = LoggerFactory.getLogger(RMHandlerTCC.class);
 
     @Override
     public void handle(UndoLogDeleteRequest request) {
         //DO nothing
     }
 
+    @Override
+    public BranchDeleteResponse handle(BranchDeleteRequest request) {
+        if (LOGGER.isDebugEnabled()) {
+            LOGGER.debug("Start tcc delete branch fence, xid:{}, branchId:{}",
+                    request.getXid(), request.getBranchId());
+        }
+        BranchDeleteResponse branchDeleteResponse = new BranchDeleteResponse();
+        try {
+            boolean result = DefaultCommonFenceHandler.get().
+                    deleteFenceByXidAndBranchId(request.getXid(), request.getBranchId());
```

Comment:
我认为tcc的删除不能简单删除，因为如tcc一阶段是预留资源，如果被删除了，理论上应该走回滚逻辑，而deletefence本身有一个定时任务会去删除，故不需要专门的deleterequest，只要下发回滚即可，如果回滚失败应该提示删除失败，在提供一个强制删除的接口，针对回滚失败的事务，强删就是只删除服务端的session数据
I believe that the deletion in TCC (Try-Confirm-Cancel) should not be a simple removal. Since TCC’s first phase reserves resources, if it's deleted, it should theoretically trigger a rollback logic. The deletefence already has a scheduled task to handle the deletion, so there’s no need for a dedicated deleterequest. Instead, a rollback should be issued. If the rollback fails, the deletion failure should be reported. Additionally, a forced deletion interface should be provided for transactions where the rollback fails. In this case, the forced deletion would only remove the session data from the server.